flink-connector/build.gradle.kts

```diff
@@ -27,7 +27,13 @@ repositories {
 }
 
 val flinkVersion: String = libs.versions.flink.get()
-val scalaVersion: String = project.properties["scalaVersion"] as? String ?: extra["defaultScalaVersion"].toString()
+
```

Comment:
@FANNG1 I kept this change to avoid the `defaultScalaVersion` changing. What do you think?

Reply:
I'm ok